src/main/resources/org/jenkinsci/plugins/github_branch_source/WildcardPullRequestLabelFilterTrait/help-includes.html

```diff
@@ -0,0 +1,29 @@
+<!--
+ ~ The MIT License
+ ~
+ ~ Copyright (c) 2017, CloudBees, Inc.
+ ~
+ ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+ ~ of this software and associated documentation files (the "Software"), to deal
+ ~ in the Software without restriction, including without limitation the rights
+ ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ ~ copies of the Software, and to permit persons to whom the Software is
+ ~ furnished to do so, subject to the following conditions:
+ ~
+ ~ The above copyright notice and this permission notice shall be included in
+ ~ all copies or substantial portions of the Software.
+ ~
+ ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ ~ THE SOFTWARE.
+ -->
+<div>
+    Space-separated list of project pull request label patterns to consider.
+    You may use <code>*</code> as a wildcard; for example: <code>jenkins *-plugin</code>
+ 
+    If the pull request has no labels it will only be included if the includes filter is <code>*</code>.
```

Comment:
```suggestion
    Space-separated list of project pull request label patterns to consider.
    You may use <code>*</code> as a wildcard; for example: <code>jenkins *-plugin</code>
    <br />
    If the pull request has any label that matches any include pattern, it will be included.
    <br />
    If the pull request has no labels it will only be included if the includes filter is <code>*</code>.
```